src/server/importWorker.js

```diff
@@ -25,6 +25,7 @@ importQueue.process(async job => {
   // Call the existing export ndjson function that writes the files
   console.log(`import-worker-${process.pid}: Kicking off export request: ${exportURL}`);
   const result = await executePingAndPull(clientEntry, exportURL, measureBundle);
+
```

Comment:
Def a nitpick, but maybe we can get rid of these line changes since there aren't any substantive changes to the importWorker?